other/edge-server/README.md

```diff
@@ -242,15 +265,15 @@ Install the required dependencies:
 npm install
 ```
 
-Add a `.env` file inside the `node-server` directory with the details 
+Add a `.env` file inside the `node-server` directory with the details
```

Comment:
Oof. I am realizing now this is even more of a PITA since you've made it generally nicer about not having to install dependencies and start the servers individually.

At the very least, since this README is now above the `client` directory and the `node-server` directory isn't visible, we should probably add that detail.

But it would really be nicer to have some higher-level thing people can config instead of having to dig deeper to configure stuff.
```suggestion
Add a `.env` file inside the `client/node-server` directory with the details
```

(We should probably also remove the lines above about cding into `node-server` and installing dependencies, since you've now got that higher-level script that manages installing all the dependencies.)

Reply:
Yeah, would be nice to streamline this more. I'll do some more thinking about a good approach in the future.